dosagelib/plugins/webtoons.py

```diff
@@ -166,6 +167,7 @@ def getmodules(cls):  # noqa: Allowed to be long
             cls('FourLeaf', 'fantasy/four-leaf', 1454),
             cls('FreakingRomance', 'romance/freaking-romance', 1467),
             cls('FridayForbiddenTales', 'thriller/friday', 388),
+            cls('GameMasters', 'challenge/game-masters',237252),
```

Comment:
```suggestion
            cls('GameMasters', 'challenge/game-masters', 237252),
```
No new warnings, please

Reply:
ah, that's what the whitespace warning was about, I was wondering.